xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1549,6 +1549,9 @@ core.recyclebin.confirm=Are you sure you wish to move this page to the recycle b
 core.recyclebin.confirmWithInlinks=Are you sure you wish to move this page to the recycle bin?
 core.recyclebin.completelyDeleteConfirm=This action is not reversible. Are you sure you wish to continue?
 core.recyclebin.invalidEntry=Invalid recycle bin entry.
+core.recyclebin.wantToSkip=Are you sure you wish to delete this page?
```

Comment:
the key name and the text don't seem to match.

Reply:
wantToSkip is not a good name to me. What does it mean?

Reply:
I'm proposing the following keys instead:
```
core.recyclebin.skipRecycleBinForm.hint=Are you sure you wish to delete this page?
core.recyclebin.skipRecycleBinForm.doNoSkipRecycleBin.label=Delete and move to the recycle bin.
core.recyclebin.skipRecycleBinForm.skipRecycleBin.label=Permanently delete the page (it won't be put in the recycle bin).
```

WDYT?

Reply:
Several problem I see:
* I don't think we should use "form" in the element name.
* I don't understand how a hint is a question so that doesn't seem good but I don't have the screenshot and I don't know where it's used.  It seems to me it's the label text. Isn't it?
* I don't get the `doNoSkipRecycleBin`
* Normally it should be `recycleBin` and not `recyclebin`. Have you kept the lowercase because it already existed?

I'd propose:

```
core.recycleBin.skip.label=Are you sure you wish to delete this page?
core.recycleBin.skip.value.yes = ...
core.recycleBin.skip.value.no = ...
core.recycleBin.noSkipWithLinks.label = ...
core.recycleBin.noSkipWithoutLinks.label = ...
```

However:
1) I haven't checked how it's named elsewhere, especially for radio buttons
2) I still don't understand what this "WithLinks" means so the last 2 key names are most likely wrong.
3) We need to decide if we use `recycleBin` (correct) or `recyclebin` (not correct but follows existing naming)

I also don't get why you use a DL/DT in one case and P in the other ones. It doesn't seem consistent.

Reply:
The UI looks like this 
![image](https://user-images.githubusercontent.com/327856/93502571-365f4a00-f917-11ea-8082-98b8175a68f0.png)

After reviewing some keys from the administration, the pattern is:
- {prefix}.fieldName for the field description
- {prefix}.fieldName.id of the radio inputs. Since id don't have ids for the inputs. I used yes/no (see https://github.com/xwiki/xwiki-platform/pull/1401/files#diff-fffbf37334c4d7a178b13f4ccc313f86R1552)

Reply:
What about the other 2 points: 
* DL (DD/DT) vs P
* The "WithLinks" question

Reply:
I see you've used uppercase B for `recycleBin`. You think it's the best? I wasn't sure since the current usage is without it. Either we go with the guideline or use what there is for consistency.

Reply:
I guess we can try to move to the norm where we can, even if this does not look like translation keys above. So I `recycleBin`.
I'm using DL (DD/DT) where a form is displayed to the user and kept P where a single information message is displayed. WDYT?
I'm not sure to understand what `WithLinks` is.

Reply:
> I'm using DL (DD/DT) where a form is displayed to the user and kept P where a single information message is displayed. WDYT?

That doesn't make sense to me. They're all a form since there's a button!

What's your rationale for the inconsistency? Also we have form best practices: https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/

Am I wrong? (it's really not my domain! :))

Reply:
Ok I see the problem, there's no best practice for standalone text in a form at https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/

So I guess the P is fine, but I'm wondering if a DL with only a DD wouldn't make sense? I don't know what's the practice TBH. What was important to me is to have a consistency in style.

Reply:
I would propose to remove the P and replace them dd/dd/dt since it is more uniform.